Deferred.xcodeproj/project.pbxproj

```diff
@@ -385,6 +391,7 @@
 			baseConfigurationReference = DB32954F1B680744002160CC /* Debug.xcconfig */;
 			buildSettings = {
 				ENABLE_TESTABILITY = YES;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
Xcode 7.1 insisted. It's in the xcconfig but it would moan every time you opened the project.